lib/liquid/standardfilters.rb

```diff
@@ -86,6 +86,7 @@ def truncatewords(input, words = 15, truncate_string = "...".freeze)
     #   <div class="summary">{{ post | split '//' | first }}</div>
     #
     def split(input, pattern)
+      pattern = pattern.to_s if pattern.is_a?(Numeric)
```

Comment:
Looks like we've said Liquid regex is not supported, so I'd vote we just remove that test and to_s https://github.com/Shopify/liquid/issues/202#issuecomment-19112872